libs/openFrameworks/events/ofEvents.h

```diff
@@ -213,6 +213,8 @@ class ofCoreEvents {
 	ofEvent<ofMouseEventArgs> 	mousePressed;
 	ofEvent<ofMouseEventArgs> 	mouseReleased;
 	ofEvent<ofMouseEventArgs> 	mouseScrolled;
+	ofEvent<void>				mouseEntered;
+	ofEvent<void>				mouseExited;
```

Comment:
GLFW only tells you whether the mouse entered or left, but we can pass along ofEvents::currentMouseX/Y. I'll add it in my next pass at this PR